### DISCUSSION TITLE
Remove sosa:isSampleOf from ANSIS entities

### DISCUSSION THREAD

**Discussion by abhritchie** [ANZSoilData/def-au-schema-json#16] [General]
While conceptually valid, `isSampleOf` appears to duplicate what some of the `has` and `related` prefixed properties do on some entities.

E.g. `SoilSite.relatedSoilBody` creates the same relation as `SoilSite.isSampleOf`.

To keep things simple in the _JSON schema_ I've removed `isSampleOf` so there is only one way to related entities. This also allows more explicit binding of particular types of entity.

**Comment by dr-shorthair** [comment]
`isSampleOf` is a very specific relationship, which says the context resource is designed to be representative of the target resource. 
`relatedX` is more general.

**Reply by abhritchie** [reply]
Makes sense.

I'll start a discussion about the role of the JSON schema and how that will impact decisions about what to keep from the ontology. Ultimately, they are complimentary things that serve different purposes. The schema isn't replacing the ontology and may not need as much conceptual rigour.
